### DISCUSSION TITLE
Is there a plugin for this?

### DISCUSSION THREAD

**Discussion by batshell** [jspsych/jsPsych#2413] [Q&A]
I want to know if there is a plugin to implement something what the https://www.moralmachine.net/ does, i.e. to show two images on the screen and you have to select any one of the two, and there's a button at the bottom that shows description.

**Comment by trig11** [comment]
You should be able to just use the html button response plugin (https://www.jspsych.org/7.0/plugins/html-button-response/) and use the images you want to show as the choices. This example below should do something quite similar to what you want to do.

```javascript
var trial = {
	type: jsPsychHtmlButtonResponse,
	stimulus: '<p>Some text here</p>',
	choices: ["<img style='border: 2px solid' src='img/orange.png'alt='orange' >", "<img style='border: 2px solid' src='img/blue.png'alt='blue' >"],
	prompt: "<p>Add your question here</p>"
};
timeline.push(trial);
```
